demisto_sdk/commands/lint/resources/pylint_plugins/xsoar_level_checker.py

```diff
@@ -34,6 +42,31 @@ def _type_annotations_checker(self, node):
             if not annotation and node.name not in ['main', '__init__']:
                 self.add_message("missing-arg-type-annoation", node=node)
 
+    def _not_implemented_error_in_main(self, node):
+        if node.name == 'main':
+            not_implemented_error_exist = False
+            for child in self._inner_search_return_error(node):
+                if isinstance(child, astroid.If):
+                    else_cluse = child.orelse
+                    for line in else_cluse:
+                        if isinstance(line, astroid.Raise) and line.exc.func.name:
```

Comment:
it just checks that an exception is raised, correct?
can we do add an str search for `NotImplementedError`?

Reply:
yes. That is what I was going for on the right side of the if , accidentally forgot to add it. Thanks for noticing